cmd/dcrdata/main.go

```diff
@@ -779,10 +779,9 @@ func _main(ctx context.Context) error {
 		r.Get("/search", explore.Search)
 		r.Get("/charts", explore.Charts)
 		r.Get("/ticketpool", explore.Ticketpool)
-		r.Get("/stats", explore.StatsPage)
 		r.Get("/market", explore.MarketPage)
-		r.Get("/statistics", func(w http.ResponseWriter, r *http.Request) {
-			http.Redirect(w, r, "/stats", http.StatusPermanentRedirect)
+		r.Get("/stats", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should it keep statistics as a redirect too so the old endpoint doesn't error?

Reply:
Errm, thought about it, but it has been there for over 6 years, enough time to keep a legacy redirect, no?

Reply:
I'm fine with removing it too.  I just thought I'd point it out since redirects are basically free and who knows if there are old links to it, etc.

No objections to removing it.